test/integration/glue_test.py

```diff
@@ -1,20 +1,136 @@
 """PyTest cases related to the integration between FERC1 & EIA 860/923."""
 import logging
 
+import pandas as pd
+import pytest
+
 from pudl.glue.ferc1_eia import (
-    get_db_plants_ferc1,
     get_mapped_plants_ferc1,
     get_mapped_utils_ferc1,
+    get_raw_plants_ferc1,
     get_unmapped_plants_eia,
     get_unmapped_plants_ferc1,
     get_unmapped_utils_eia,
-    get_unmapped_utils_ferc1,
+    get_unmapped_utils_ferc1_dbf,
+    glue,
+)
+from pudl.glue.xbrl_dbf_ferc1 import (
+    get_util_ids_ferc1_raw_xbrl,
+    get_utils_ferc1_raw_dbf,
 )
 from pudl.metadata.classes import DataSource
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="module")
+def glue_dfs() -> dict[str, pd.DataFrame]:
+    """Make a dictionary of glue dataframes."""
+    return glue(eia=True, ferc1=True)
+
+
+@pytest.fixture(scope="module")
+def utilities_pudl(glue_dfs) -> pd.DataFrame:
+    """A table of PUDL utilities IDs."""
+    return glue_dfs["utilities_pudl"]
+
+
+@pytest.fixture(scope="module")
+def utilities_ferc1(glue_dfs) -> pd.DataFrame:
+    """A table of FERC1 utilities IDs."""
+    return glue_dfs["utilities_ferc1"]
+
+
+@pytest.fixture(scope="module")
+def utilities_ferc1_xbrl(glue_dfs) -> pd.DataFrame:
+    """A table of FERC1 XBRL utilities IDs."""
+    return glue_dfs["utilities_ferc1_xbrl"]
+
+
+@pytest.fixture(scope="module")
+def utilities_ferc1_dbf(glue_dfs) -> pd.DataFrame:
+    """A table of FERC1 DBF utilities IDs."""
+    return glue_dfs["utilities_ferc1_dbf"]
```

Comment:
I assume there is a way to parameterize the creation of these fixtures....?

Reply:
Rather than creating a separate fixture for each of these where you use a string to select a dataframe from a dictionary of dataframes, it seems like you could just have the one `glue_dfs` fixture, and use the keys (strings) to look up the dataframes directly as the parameters in the test below. Then you don't have to do the `getfixturevalue()` thing either.

Reply:
yeah that feels cleaner code wise but it does require ALL of the tables to be created basically for any one of the tests (i.e. all the input tables will need to be built before any of the actual test are run. I still think this is the way to go... but it does couple everything in the module.